src/transformers/models/gemma/modeling_gemma.py

```diff
@@ -170,7 +173,16 @@ def __init__(self, config):
         self.gate_proj = nn.Linear(self.hidden_size, self.intermediate_size, bias=False)
         self.up_proj = nn.Linear(self.hidden_size, self.intermediate_size, bias=False)
         self.down_proj = nn.Linear(self.intermediate_size, self.hidden_size, bias=False)
-        self.act_fn = ACT2FN[config.hidden_act]
+        hidden_act = config.hidden_act
+        if hidden_act != "gelu_pytorch_tanh":
```

Comment:
This way even if there is a model with the old gelu on the config we're force-setting hidden_act to `"gelu_pytorch_tanh"` right? 
I think we should either use a new config name or create a new attribute in the config `force_use_exact_gelu`, that is iniailizaed to False so that users can have the flexibility to switch to the old act function in case they fine-tuned it with old GeLU, what do you think?

Reply:
Hmm I like that approach ie `getattr(config, "force_use_exact_gelu", False)` so if `force_use_exact_gelu = True` then True. If `force_use_exact_gelu = False` then also False, and False otherwise.

Reply:
yes! I think we can add that directly into `GemmaConfig` class and default it to False

Reply:
I added `force_use_exact_gelu`!